ydb/library/actors/wilson/wilson_uploader.cpp

```diff
@@ -36,26 +110,18 @@ namespace NWilson {
             NServiceProto::ExportTraceServiceResponse Response;
             grpc::Status Status;
 
-            struct TSpanQueueItem {
-                TMonotonic ExpirationTimestamp;
-                NTraceProto::Span Span;
-                ui32 Size;
-            };
-
-            std::deque<TSpanQueueItem> Spans;
+            TBatch CurrentBatch;
+            std::queue<TBatch::TData> BatchQueue;
             ui64 SpansSize = 0;
```

Comment:
It's better to rename it -- a few lines above SizeSpans means items, here it means bytes.

Reply:
Renamed to SizeSpansBytes